README.md

```diff
@@ -1375,36 +1375,36 @@ balancer, site‐to‐site VPN, and NAT services.
 * `use_external_resource`: (optional) Use external object. The default is `false`.
 * `resource_id`: (optional) [NSX object ID](README.md#resource_id), used to identify the object when `use_external_resource` is `true`.
 * `edge`:
-  * `name`: The name of the ESG to be created.
-  * `esg_pwd`: The CLI password of the ESG.
-  * `esg_size`: The size of the ESG, possible values: `compact`, `large`, `quadlarge`, `xlarge`.
-  * `datacentermoid`: The [vCenter DataCenter ID](README.md#resource_id) where dlr control vm will be deployed.
-  * `datastoremoid`: The [vCenter DataStore ID](README.md#resource_id) where dlr control vm will be deployed.
-  * `resourcepoolid`: The [vCenter Cluster ID](README.md#resource_id) where dlr control vm will be deployed.
-  * `default_pg`: The managed object id of the port group for the first vnic (on creation the first vnic must be connected to a valid portgroup in NSX).
+  * `name`: The name of the [ESG](README.md#cloudifynsxesg) to be created.
+  * `esg_pwd`: The CLI password of the [ESG](README.md#cloudifynsxesg).
+  * `esg_size`: The size of the [ESG](README.md#cloudifynsxesg), possible values: `compact`, `large`, `quadlarge`, `xlarge`.
+  * `datacentermoid`: The [vCenter DataCenter ID](README.md#resource_id) where the [DLR](README.md#cloudifynsxdlr) control `VM` will be deployed.
+  * `datastoremoid`: The [vCenter DataStore ID](README.md#resource_id) where the [DLR](README.md#cloudifynsxdlr) control `VM` will be deployed.
+  * `resourcepoolid`: The [vCenter Cluster ID](README.md#resource_id) where the [DLR](README.md#cloudifynsxdlr) control `VM` will be deployed.
+  * `default_pg`: The managed object id of the port group for the first vnic (on creation the first `VNIC` must be connected to a valid portgroup in NSX).
```

Comment:
~The managed object id of the port group for the first vnic~ The managed object **ID** of the port group for the first **VNIC** ...